docs/configuration.md

```diff
@@ -46,45 +46,46 @@ Iceberg tables support table properties to configure table behavior, like the de
+| write.spark.fanout.enabled                     | false                      | Enables the fanout writer in Spark that does not require data to be clustered; uses more memory                                                                                                   |
+| write.object-storage.enabled                   | false                      | Enables the object storage location provider that adds a hash component to file paths                                                                                                             |
+| write.data.path                                | table location + /data     | Base location for data files                                                                                                                                                                      |
+| write.metadata.path                            | table location + /metadata | Base location for metadata files                                                                                                                                                                  |
+| write.delete.mode                              | copy-on-write              | Mode used for delete commands: copy-on-write or merge-on-read (v2 only)                                                                                                                           |
+| write.delete.isolation-level                   | serializable               | Isolation level for delete commands: serializable or snapshot                                                                                                                                     |
+| write.update.mode                              | copy-on-write              | Mode used for update commands: copy-on-write or merge-on-read (v2 only)                                                                                                                           |
+| write.update.isolation-level                   | serializable               | Isolation level for update commands: serializable or snapshot                                                                                                                                     |
+| write.merge.mode                               | copy-on-write              | Mode used for merge commands: copy-on-write or merge-on-read (v2 only)                                                                                                                            |
+| write.merge.isolation-level                    | serializable               | Isolation level for merge commands: serializable or snapshot                                                                                                                                      |
```

Comment:
Can you just update the rows needed for this change? We try to avoid reformatting tables unless they're really messy.

Reply:
Sure this is done (my editor got a little aggressive)